wpeverest/everest-forms
# Exclude compiled and minified JS and CSS assets from Git

### All Submissions:






### Changes proposed in this Pull Request:

As discussed on numerous occasions and recently amongst the team, this PR removes compiled and minified CSS and JS assets from Git.

The reasoning is to mainly eliminate merge conflicts, but also make it easier to contribute JS and CSS without too much confusion.

When developing and making CSS or JS changes one always needs to have `SCRIPT_DEBUG` set to true as well as run `grunt watch` this will ensure that changes are shown when made.

When committing changes to CSS only the SCSS file changes must be committed, for JS only changes to the .js file must be committed and not the .min.js file.

Our deploy script should be updated to include building these assets and committing to SVN when deploying to wp.org. The only other process that may need attention is releases on GitHub, but if I recall that is also handled via our release script so should be fine cc @rabinstha 

Would love some additional feedback from @rabinstha before merging this to see if there are any other angles that need to be covered.

### How to test the changes in this Pull Request:

1. Checkout this branch and ensure that there are no *.min.js files in assets/js/admin and assets/js/frontend. Also check assets/css folder does not contain any *.css files.
2. Now run `grunt assets` this should compile and minify all the css and js files
3. Try a git commit, no files should be available for commit.

### Other information:
